multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -911,6 +912,7 @@ def create_transaction_for_changing_token_to_dynamic(self,
     def create_transaction_for_updating_token_id(self,
                                                  sender: IAddress,
                                                  token_identifier: str) -> Transaction:
+        """The following token types cannot be changed to dynamic: FungibleESDT, NonFungibleESDT, NonFungibleESDTv2"""
```

Comment:
this is not correct here for `updateTokenID`

Reply:
it's taken from the docs:
https://docs.multiversx.com/tokens/nft-tokens/#update-token

Reply:
it is wrong in docs, there is already an open PR for this https://github.com/multiversx/mx-docs/pull/1003/files